pkg/netdevice/rdma.go

```diff
@@ -16,6 +19,11 @@ type rdmaSpec struct {
 func NewRdmaSpec(pciAddrs string) types.RdmaSpec {
 	deviceSpec := make([]*pluginapi.DeviceSpec, 0)
 	isSupportRdma := false
+
+	if _, err := os.Stat("/dev/infiniband"); err != nil {
+		glog.Warning("dir \"/dev/infiniband\" is not found")
```

Comment:
what may cause this issue ? i think its when rdma modules are not loaded right ?

perhaps the warning could be: "dir /dev/infiniband is not found.  are RDMA modules loaded?" or something similar.
what do you think ?

it should be noted that the _"infiniband"_ in `/dev/infiniband` is just legacy name and also refers to chardevs created for RDMA over other fabrics as well e.g ethernet  (RoCE)

Reply:
updated the warning message